doge_datagen/kafka_sink_factory.py

```diff
@@ -61,19 +61,15 @@ def close(self):
 
 class KafkaSinkFactory(object):
     def __init__(self,
-                 bootstrap_servers: Iterable[str],
-                 client_id: str,
-                 buffer_size=100000):
+                conf: dict,
```

Comment:
Formatting: please align it with other parameters

Reply:
Done